### DISCUSSION TITLE
Need help with Github Teams plan with Repository rules feature for our organisation

### DISCUSSION THREAD

**Discussion by parteek-s** [community/community#114629] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,
   I am planing to purchase  github's **Teams** plan for our organisation that would be cost 3.67$ user/month. But I also need **Repository rule** feature with this plan. So, can we get **Repository rule** feature with **Teams** plan? If yes, then what would be the cost for this..

   If not then can we have some discount on **Enterprise** plan?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @parteek-s ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
